assets/js/course.js

```diff
@@ -1,4 +1,8 @@
-jQuery(document).ready( function($) {
+jQuery( document ).ready( function ( $ ) {
```

Comment:
You can ignore this file, since I just run the formatter for JS and it was updated.